services/core-web/.eslintrc.js

```diff
@@ -39,5 +39,6 @@ module.exports = {
     camelcase: 0,
     "jsx-a11y/label-has-for": 0, // deprecated rule
     "import/no-cycle": 0,
+    "no-param-reassign": 0,
```

Comment:
I am happy to get rid of this one, finally! 😆

Reply:
haha saaame!